src/Routes.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { createStackNavigator } from "@react-navigation/stack";
+import { NavigationContainer } from "@react-navigation/native";
+import { Button, Text, View } from 'react-native';
+import { Center } from './Center';
+
+interface RoutesProps{}
+
+const Stack = createStackNavigator();
+
+function Home({ navigation }){
```

Comment:
This code is defining a new component. You should pull this out into a new file.

You could create a folder called `views` or `pages` and put the components that you pass to your router in that folder.
Ex:
```
src > pages
  Home.tsx
  Workout.tsx
```

Then define your components in those files using the same syntax you defined the Routes component:
```javascript
export const Home: React.FC = ({ navigation }) => {
  return (
    // ... component template code here
  )
}
```